example/classifier/odp_classifier.c

```diff
@@ -666,6 +694,10 @@ int main(int argc, char *argv[])
 		}
 	}
 
+	if (check_ci_pass_count(args)) {
+		ODPH_ERR("Error: Packet count verification failed\n");
+		exit(EXIT_FAILURE);
+	}
```

Comment:
A better place for this check would be couple lines below after the `odph_thread_join(thread_tbl, num_workers);` line.

Reply:
Ack